kv/dist_sender.go

```diff
@@ -518,7 +518,10 @@ func (ds *DistSender) sendChunk(ctx context.Context, ba roachpb.BatchRequest) (*
 	// Local addressing has already been resolved.
 	// TODO(tschottdorf): consider rudimentary validation of the batch here
 	// (for example, non-range requests with EndKey, or empty key ranges).
-	rs := keys.Range(ba)
+	rs, rErr := keys.Range(ba)
```

Comment:
All of these variables should be named `err`. We sometimes use other names when it's a different type (e.g. `pErr` for `roachpb.Error`) or in special cases, but most of the time a variable of type `error` should be named `err`.